### PR TITLE
Create 0000-01-02-UncleSamJCU.md

### PR DESCRIPTION
**Pull request by UncleSamJC**
added a file, pls look

### CODE REVIEW

File: _posts/_posts/0000-01-02-UncleSamJCU.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (UncleSamJCU) [inline _posts/_posts/0000-01-02-UncleSamJCU.md:5]: My text

**Review by UncleSamJC — COMMENTED** [review]
Finish review

File: _posts/_posts/0000-01-02-UncleSamJCU.md (hunk 1 above)
Comment (UncleSamJCU) [inline _posts/_posts/0000-01-02-UncleSamJCU.md:5]: +++ text

**Review by UncleSamJC — COMMENTED** [review]
END

**Pull request merged**